dm/pkg/binlog/event/util.go

```diff
@@ -408,6 +408,26 @@ func GetCharsetCodecByStatusVars(statusVars []byte) (encoding.Encoding, error) {
 	}
 }
 
+// GetTimezoneByStatusVars returns the timezone of upstream for "datetime" type alter table commands.
+// For "ALTER TABLE ADD COLUMN x DATETIME", not for "ALTER TABLE ADD COLUMN x TIMESTAMP".
+func GetTimezoneByStatusVars(statusVars []byte, upstreamTZStr string) (string, error) {
+	vars, err := statusVarsToKV(statusVars)
+	b, _ := vars[QTimeZoneCode]
```

Comment:
didn't check whether `QTimeZoneCode` exist. previous impl is ok, but maybe need add some comments about why we didn't check error first as usual.

as of `undocumented vars`, it's related to `statusVarsToKV`, we can refactor it later.

Reply:
Understand. Added the comment.